File: ProgramasPython/secao03/Exercicio06.py
horas   =   int(input("Digite o total de horas que você trabalhou neste mês: "));
valor   =   float(input("Informe o valor ganho por hora trabalhada: "));
salario =   (horas * valor);

print("O salário ganho neste mês é de R${0:.2f}.".format(salario));

File: ProgramasJava/src/br/com/geekuniversity/secao03/Exercicio06.java
package br.com.geekuniversity.secao03;

import java.util.*;

public class Exercicio06 {

	public static void main(String[] args) {
		
		int hrs;
		float valor, salario;
		Scanner tela = new Scanner(System.in);
				
		System.out.println("Informe o valor do salário ganho por hora: ");
		valor = tela.nextFloat();
		
		System.out.println("Informe o total de horas trabalhadas no mês: ");
		hrs = tela.nextInt();
		
		salario = (valor * hrs);
		
		System.out.println("Este mês, o salário recebido será de R$" + salario);
		
		tela.close();
		
	}

}
